api/resources/items.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+import json
+
+import bleach
+from flask import request
+from flask_restful import Resource, abort
+from sqlalchemy.orm.exc import NoResultFound
+
+from api import db
+from api.database.models import Item
+
+
+# def _user_payload(user):
```

Comment:
Nice to keep as a reference and we can delete it once we have a better idea of what we're doing. :)